stages/org.osbuild.sfdisk

```diff
@@ -135,14 +141,15 @@ class PartitionTable:
         command = f"label: {self.label}\nlabel-id: {self.uuid}"
         for partition in self.partitions:
             fields = []
-            for field in ["start", "size", "type", "name", "uuid"]:
+            for field in ["start", "size", "type", "name", "uuid", "attrs"]:
                 value = getattr(partition, field)
                 if value:
                     fields += [f'{field}="{value}"']
             if partition.bootable:
                 fields += ["bootable"]
             command += "\n" + ", ".join(fields)
 
+        print("********************************************************************")
```

Comment:
Left-over debugging message? ;)

Reply:
Oops, yes, is a left-over. I'm going to address the comments and send a second version. Thanks for the review.